rules/cross-platform/credential_access_tcpdump_activity.toml

```
@@ -21,14 +21,15 @@ index = ["auditbeat-*", "logs-endpoint.events.*"]
 language = "kuery"
 license = "Elastic License"
 name = "Network Sniffing via Tcpdump"
+note = "For macOS systems, Auditbeat must be run with an account that has access to the binaries in the /usr/sbin, /sbin, /usr/bin, and /bin directories. For Linux systems, to avoid conflicts with auditd, you will need to stop and disable auditd with 'sudo service stop auditd && sudo systemctl disable auditd'."
 risk_score = 21
 rule_id = "7a137d76-ce3d-48e2-947d-2747796a78c0"
 severity = "low"
-tags = ["Elastic", "Linux"]
+tags = ["Elastic", "Linux", "macOS"]
 type = "query"
 
 query = '''
-event.category:process and event.type:(start or process_started) and process.name:tcpdump
+event.category:process and process.name:tcpdump and (event.action:executed or event.type:start)
```

do you mind switching? just makes it easier to read hierarchically (context [CONTRIBUTING.md#using-elastic-common-schema-ecs](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md#using-elastic-common-schema-ecs))

```suggestion
event.category:process and (event.action:executed or event.type:start) and process.name:tcpdump 
```